shared-libs/infodoc/test/infodoc.js

```diff
@@ -503,7 +481,7 @@ describe('infodoc', () => {
 
   describe('recordDocumentWrites', () => {
     describe('update one', () => {
-      let sentinelGet; let 
+      let sentinelGet; let
```

Comment:
Nit: because you're changing this anyway, move the `let` down a line to fix the formatting fully. Same with line 564. I assume this was done automatically by your editor?

Reply:
Yeah, my editor is configured to [not save trailing whitespace](https://github.com/medic/medic-docs/blob/master/development/style-guide.md#no-trailing-whitespace). This weird formatting wasn't done because of that though if that's what you mean, that was in an [eslint refactor of all things](https://github.com/medic/cht-core/commit/2c1db1618bdafc5ec14c3a27aa4a37249fcc1b4a#diff-663b057635d166a6ae6c764fc757973dR586-R587).

But yep, found and fixed